spikeextractors/extractors/cedextractors/utils.py

```diff
@@ -70,12 +73,17 @@ def get_channel_data(f, smrx_ch_ind, start_frame=0, end_frame=None):
     if end_frame is None:
         end_frame = int(f.ChannelMaxTime(smrx_ch_ind) / f.ChannelDivide(smrx_ch_ind))
 
+    nMax = int(f.ChannelMaxTime(smrx_ch_ind) / f.ChannelDivide(smrx_ch_ind))
+    frame_offset = int(f.FirstTime(chan=smrx_ch_ind, tFrom=0, tUpto=nMax) / f.ChannelDivide(smrx_ch_ind))
```

Comment:
I think this is strange. For me tUpTo is in ticks so without device.

Reply:
See This https://github.com/samuelgarcia/python-neo/blob/5321483836a72e853a8f4812e796cb635cbdef77/neo/rawio/cedrawio.py#L161

Reply:
I agree it is strange, but this arg seems to work as a max number of frames. You can test it by fixing a small value there (e.g. 10) and you'll see that it will return max 10 frames